tachikoma-postfix-utils/src/main/kotlin/com/sourceforgery/tachikoma/postfix/Main.kt

```diff
@@ -30,35 +27,17 @@ internal constructor(
     fun run() {
         LOGGER.info { "Connecting to ${tachikomaUrl.withoutPassword()}" }
 
-        val plaintext = tachikomaUrl.scheme == "http"
-
-        val metadataAuth = Metadata()
-        metadataAuth.put(APITOKEN_HEADER, tachikomaUrl.userInfo)
+        val builder = Clients.builder(tachikomaUrl.withoutPassword().ensureGproto())
```

Comment:
Builder is much nicer!

Reply:
Yeah, and no ****** netty dependency-hell